extensions/VisualEditor/wikia/modules/ve/ui/pages/ve.ui.WikiaParameterPage.js

```diff
@@ -6,6 +6,104 @@
  */
 
 /**
+ * Restores infobox dialog info when going back from media dialog
+ *
+ * @param {ve.ui.WindowManager} win
+ * @param {ve.dm.WikiaTransclusionModel} model - original tranclusion model of infobox dialog
+ * @param {string} imageName - image name from media dialog
+ * @param {string} paramName - Name of a field (e.g. image1)
+ */
+function restoreInfoboxDialog(win, model, imageName, paramName) {
+	if (imageName) {
+		model.parts[0].params[paramName].value = imageName;
+	}
+
+	win.transclusionModel = model;
+
+	win.bookletLayout.$element.html('');
+	win.initializeTemplateParameters();
+}
+
+/**
+ * shorthand function to open dialogs
+ *
+ * @param {string} name - Name of a dialog to open
+ */
+
+function openDialog(name) {
+	ve.ui.actionFactory
+		.create('window', ve.init.target.getSurface())
+		.open(name);
+}
+/**
+ *
+ * @param {ve.ui} currentWindow
+ * @param {ve.dm.WikiaCart} cartItems
+ * @returns {string|null}
+ */
+function getImageName(currentWindow, cartItems) {
+	if (
+		currentWindow.currentAction &&
+		currentWindow.currentAction.getAction() === 'goback' &&
```

Comment:
Why do we need to check the action? This function seems to be called only when we need it anyway.